django/db/models/lookups.py

```diff
@@ -215,6 +215,10 @@ class Transform(RegisterLookupMixin, Func):
     def lhs(self):
         return self.get_source_expressions()[0]
 
+    def process_lhs(self, compiler, connection, lhs=None):
+        lhs = lhs or self.lhs
+        return compiler.compile(lhs)
+
```

Comment:
For those who were wondering `lhs=None` is completely unused internally for `Transform` subclasses.

It's done this way to maintain symmetry with `Lookup.process_lhs` which was the main motivating factor for this ticket in the first place.